examples/file_parser.rs

```diff
@@ -61,7 +61,7 @@ fn main() {
     let res = webrtc_sdp::parse_sdp(&s, true);
     match res {
         Err(why) => panic!("Failed to parse SDP with error: {}", why),
```

Comment:
Clippy isn't complaining about the `why` not being inside the panic message?

Reply:
That is a good question ...

Reply:
@nils-ohlmeier apparently `panic!` and `format!` have different string interpolation concerns:
```
warning: panic message contains unused formatting placeholders
  --> examples/file_parser.rs:29:44
   |
29 |         Err(why) => panic!("Failed to open {display}: {why}"),
   |                                            ^^^^^^^^^  ^^^^^
   |
   = note: this message is not used as a format string when given without arguments, but will be in Rust 2021
   = note: `#[warn(non_fmt_panics)]` on by default
help: add the missing arguments
   |
29 |         Err(why) => panic!("Failed to open {display}: {why}", ...),
   |                                                             +++++
```

Reply:
Then we will wait till Rust 2021 I guess :-)